extensions/adobe/experience/journeyOrchestration/stepEvents/journeyOrchestrationProfileFieldsMixin.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/journeyIdentity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration Identity Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/"
+  ],
+  "description": "This event is in relation with journey and describe the profile Identifiers",
+  "definitions": {
+    "journeyIdentity": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/profile": {
+          "title": "Journey Orchestration Identity Fields.",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/journeyOrchestration/profile/ID": {
+              "title": "Profile ID",
+              "type": "string",
+              "description": "Profile identifier identifies the profile sent/used in a journey.Present if the current step is running in the context of a unitary journey. E.g: foo@adobe.com."
```

Comment:
This description can be confusing as for consultants unitary journeys are journeys in which Read Segment is not there. But we have these fields even in those journeys. So we can simply omit the last line here.